.storybook/seed-fake.js

```diff
@@ -468,6 +468,42 @@ export const seedFake = (db) => {
     errors: [],
   })
 
+  const device7 = db.addDevice({
+    workspace_id: ws2.workspace_id,
+    connected_account_id: ca.connected_account_id,
+    device_type: 'noiseaware_activity_zone',
+    created_at: '2023-05-15T15:08:53.000',
+    name: 'Living Room',
+    properties: {
+      online: true,
+      model: {
+        manufacturer_display_name: 'NoiseAware',
+        display_name: 'Indoor Sensor',
+      },
+    },
+    errors: [],
+  })
+
+  db.addNoiseThreshold({
+    device_id: device7.device_id,
+    workspace_id: ws2.workspace_id,
+    created_at: '2023-05-17T00:16:12.000',
+    noise_threshold_decibels: 70,
+    starts_daily_at: '22:00:00[America/Los_Angeles]',
+    ends_daily_at: '06:00:00[America/Los_Angeles]',
+    name: 'builtin_quiet_hours',
+  })
+
+  db.addNoiseThreshold({
+    device_id: device7.device_id,
+    workspace_id: ws2.workspace_id,
+    created_at: '2023-05-17T00:16:12.000',
+    noise_threshold_decibels: 75,
+    starts_daily_at: '06:00:00[America/Los_Angeles]',
+    ends_daily_at: '22:00:00[America/Los_Angeles]',
+    name: 'builtin_quiet_hours',
+  })
```

Comment:
The underscore names look a little odd in the storybook. Can we do this? 

```suggestion
  db.addNoiseThreshold({
    device_id: device7.device_id,
    workspace_id: ws2.workspace_id,
    created_at: '2023-05-17T00:16:12.000',
    noise_threshold_decibels: 70,
    starts_daily_at: '22:00:00[America/Los_Angeles]',
    ends_daily_at: '06:00:00[America/Los_Angeles]',
    name: 'Quiet Hours',
  })

  db.addNoiseThreshold({
    device_id: device7.device_id,
    workspace_id: ws2.workspace_id,
    created_at: '2023-05-17T00:16:12.000',
    noise_threshold_decibels: 75,
    starts_daily_at: '06:00:00[America/Los_Angeles]',
    ends_daily_at: '22:00:00[America/Los_Angeles]',
    name: 'Active Hours',
  })
```